app/controllers/weeks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class WeeksController < ApplicationController
+  before_action :authenticate_user!
+
+  # GET /weeks
+  def index
+    current_user.picks.unevaluated.each(&:evaluate)
+
+    week_number = params[:week] ? params[:week] : current_week
+  end
+
+  # GET /weeks/1.json
+  def show
+    week_number = params[:id]
+    render :json => Game.where(week: week_number)
+  end
+
+  private
+    # Use callbacks to share common setup or constraints between actions.
+    def current_week
+      today = Date.today
+      last_day = Date.new(2018, 2, 22)
+      first_day = Date.new(2017, 8, 22)
+      
+      return if today > last_day || today < first_day
+      ((today - first_day)).to_i/7-1
+    end
+  # end private
+end
```

Comment:
I probably need to spend more time with the Vue code to really get why you're deprecating the games controller in favor the weeks controller, but in the Rails world this is pretty weird. Rails controllers generally should be named after the models they allow you to access. The week is just a filter. If you really want the url to be `/week` I think that's something we can handle in the router and then just point it to the appropriate action in the games controller.

Reply:
I attribute it to overzealous naming on behalf of my mental modeling. Now that it's done, we can definitely revert it back to the Rails world mode.